### ISSUE TITLE
[v1.1-Elements-Media.Sources.json]: After activating 'video' button, Audio button and controls of Video is not accessible by voice over.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7021]
https://microsoft.visualstudio.com/OS/_workitems/edit/37844142
